tests/distributed/test_partition.py

```diff
@@ -180,7 +180,12 @@ def check_hetero_partition(hg, part_method, num_parts=4, num_trainers_per_machin
     shuffled_labels = []
     shuffled_elabels = []
     for i in range(num_parts):
-        part_g, node_feats, edge_feats, gpb, _, ntypes, etypes = load_partition('/tmp/partition/test.json', i)
+        part_g, node_feats, edge_feats, gpb, _, ntypes, etypes = load_partition(
+            '/tmp/partition/test.json', i, load_feats=load_feats)
+        if not load_feats:
+            assert not node_feats
+            assert not edge_feats
+            node_feats, edge_feats = load_partition_feats('/tmp/partition/test.json', i)
```

Comment:
How about
```python
        if not load_feats:
            assert not node_feats
            assert not edge_feats
        else:
            node_feats1, edge_feats1 = load_partition_feats('/tmp/partition/test.json', i)
            # ... verify node/edge_feats1 and node/edge_feats are the same
```

Reply:
I'd like to test `load_partition(..., load_feats=False)` as well. So I think current one is more suitable and the correctness of `node/edge_feats` is verified in below.